content/help/faq/security/check-policy.md

```diff
@@ -0,0 +1,14 @@
+---
+title: How can I check whether mutual TLS is enabled for a service?
+weight: 11
+---
+
+Istioctl provides an 'authn' option to this purpose. You can do:
+
+{{< text bash >}}
+$ istioctl authn tls-check httpbin.default.svc.cluster.local
+HOST:PORT                                  STATUS     SERVER     CLIENT     AUTHN POLICY        DESTINATION RULE
+httpbin.default.svc.cluster.local:8080     OK         mTLS       mTLS       default/            default/default
+{{< /text >}}
+
+Refer to [Verifying TLS configuration](/docs/tasks/security/mutual-tls/#verifying-tls-configuration) for more information.
```

Comment:
this is a bad link - but probably dependent on the other reorg taking place in the security section.

Reply:
This depends on https://github.com/istio/istio.github.io/pull/1972/files , i will rebase and ping back once that PR is merged.